src/Controllers/AccountController.cs

```diff
@@ -0,0 +1,96 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using server.Context;
+using server.Dtos.Account;
+using server.Models;
+using server.Responses.Account;
+
+namespace Server.Controllers;
+
+public class AccountController : ApiControllerBase
```

Comment:
please use the new `Primary Constructor` feature of `dotnet 8` it looks much cleaner for DI

Reply:
I will upgrade `dotnet` to `8` in other `PR`